### PR TITLE
SG-34295 - Rebranding

### PR DESCRIPTION
**Pull request by juanburgosautoglb**
Change all the shotgrid/shotgun occurrences in the repository.

### CODE REVIEW

File: flame_assist_icon_256.png
Comment (julien-lang) [inline flame_assist_icon_256.png:1]: Problem here!!

File: icon_256.png
Comment (julien-lang) [inline icon_256.png:1]: Problem here!

**Review by julien-lang — CHANGES_REQUESTED** [review]
A couple of issues here.

Also, what about the following code in flame_hooks/sg_hook.py: 
```python
    name = (
        "Shotgun"
        if version_major < 2022 or version_major == 2022 and version_minor <= 0
        else "ShotGrid"
    )
```

Should that be updated too @jpbrault?

**Comment by jpbrault** [comment]
> A couple of issues here.
> 
> Also, what about the following code in flame_hooks/sg_hook.py:
> 
> ```python
>     name = (
>         "Shotgun"
>         if version_major < 2022 or version_major == 2022 and version_minor <= 0
>         else "ShotGrid"
>     )
> ```
> 
> Should that be updated too @jpbrault?

This need to match what is in the Flame code base.
I assume there will be a matching PR in the Flame code base, however it need to match already realeased version as well.

**Review by jpbrault — APPROVED** [review]

**Review by julien-lang — APPROVED** [review]
LGTM!

**Pull request merged**